xen-ringwatch: Update sysfs path to work with both classic and PVOPS based kernels

diff --git a/tools/misc/xen-ringwatch b/tools/misc/xen-ringwatch
index b81db77..60e60ed 100644
--- a/tools/misc/xen-ringwatch
+++ b/tools/misc/xen-ringwatch
@@ -55,7 +55,7 @@ class Pattern(object):
 class XenBackend(object):
     """A Xen I/O backend."""
 
-    SYSFS_BASEDIR = "/sys/devices/xen-backend"
+    SYSFS_BASEDIR = "/sys/bus/xen-backend/devices"
 
     def __init__(self, rd, devid):
         self.rd    = int(rd)
